WordPress/Classes/Services/PostService.h

```diff
@@ -32,6 +32,18 @@ extern NSString * const PostServiceTypeAny;
                     success:(void (^)())success
                     failure:(void (^)(NSError *error))failure;
 
+- (void)syncPostsOfType:(NSString *)postType
+           withStatuses:(NSArray *)postStatus
+                forBlog:(Blog *)blog
+                success:(void (^)(BOOL hasMore))success
+                failure:(void (^)(NSError *))failure;
+
+- (void)loadMorePostsOfType:(NSString *)postType
```

Comment:
This particular method made me wonder (without peeking at it's code) what `load` means.  I suggest documenting at least this one to detail what it does and why it would be used.